backend/hitas/models/ownership.py

```diff
@@ -27,3 +52,37 @@ class Meta:
 
     def __str__(self):
         return f"{self.owner}, {self.apartment}"
+
+
+class _OwnershipLike(TypedDict, total=False):
+    start_date: Optional[datetime.date]
+    end_date: Optional[datetime.date]
+
+
+class OwnershipLike(_OwnershipLike):
+    percentage: Decimal
+    owner: "Owner"
+
+
+def check_ownership_percentages(ownerships: Iterable[OwnershipLike]) -> None:
+    for ownership in ownerships:
+        percentage = ownership["percentage"]
+        if not 0 < percentage <= 100:
+            raise ValidationError(
+                {
+                    "percentage": (
+                        f"Ownership percentage greater than 0 and"
+                        f" less than or equal to 100. Given value was {percentage}."
+                    )
```

Comment:
I know it's copypasted but the sentence has a typo 😅 
=> `Ownership percentage must be greater than 0 and`

Reply:
Done